src/internal/connector/onedrive/permission.go

```diff
@@ -173,6 +176,7 @@ func diffPermissions(
 // the necessary permissions on onedrive objects.
 func restorePermissions(
 	ctx context.Context,
+	m365 account.M365Config,
```

Comment:
nit: (just for consistency)
```suggestion
	creds account.M365Config,
```